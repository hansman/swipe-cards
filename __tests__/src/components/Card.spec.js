import 'react-native';
import React from 'react';
import Card from '../../../src/components/Card.react';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
      <Card />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
