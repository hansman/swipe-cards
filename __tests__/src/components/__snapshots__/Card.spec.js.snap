exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderColor": "rgba(0, 0, 0, 0.2)",
      "borderRadius": 4,
      "borderWidth": 1,
      "height": 1174,
      "justifyContent": "center",
      "marginLeft": 195,
      "marginTop": 30,
      "shadowColor": "rgba(0, 0, 0, 0.5)",
      "shadowOffset": Object {
        "height": 1,
        "width": 2,
      },
      "shadowOpacity": 1,
      "width": 360,
    }
  }>
  <Image
    onError={[Function]}
    resizeMode="contain"
    source={
      Object {
        "uri": "undefinedasdf",
      }
    }
    style={
      Object {
        "borderRadius": 5,
        "height": 320,
        "width": 320,
      }
    } />
</View>
`;
